Makefile

```
@@ -58,7 +80,7 @@ endif
 
 # Setting SHELL to bash allows bash commands to be executed by recipes.
 # Options are set to exit when a recipe line exits non-zero or a piped command fails.
-SHELL = /usr/bin/env bash -o pipefail
+SHELa = /usr/bin/env bash -o pipefail
```

Some weird typo here...

```suggestion
SHELL = /usr/bin/env bash -o pipefail
```

---

nice catch thanks, no clue how this happened.